package.json

```diff
@@ -343,6 +344,11 @@
                 "category": "PostgreSQL",
                 "command": "cosmosDB.getPostgresCredentials",
                 "title": "Enter Server Credentials"
+            },
+            {
+                "category": "PostgreSQL",
+                "command": "cosmosDB.configurePostgresFirewall",
+                "title": "Configure Firewall"
```

Comment:
I like how you added this to the command palette, but I would add a confirmation message for this entry point. I think it's a good idea to make sure people didn't mis-click the wrong command/server before we change something security related. Plus it's nice to be able to see your current IP in the message

Reply:
In terms of the `GenericTreeItem` entry point, I'm somewhat indifferent on if that has the confirmation message. I feel it's not entirely necessary, but it might be simpler code-wise if they just both use it. If you do that, maybe change the message in the tree to something simpler and save the extra info (like my current IP) for the confirmation message. Something like this which would match the credentials message:
`Configure firewall to connect to "emjpg"...`